### ISSUE TITLE
Some options are not available from command line

### ISSUE THREAD

**Issue by mociepka** [Modernizr/Modernizr#2040]
I started with:

```bash
$ ./node_modules/.bin/modernizr --features touchevents --options addTest,prefixes,setClasses,testStyles
```

This command returns:

```
node_modules/modernizr/bin/modernizr:125
          throw new Error('invalid key value name - ' + prop);
          ^

Error: invalid key value name - setClasses
    at /home/chf/workspace/mirumee/prettygreen/node_modules/modernizr/bin/modernizr:125:17
    at Array.map (native)
    at find (/home/chf/workspace/mirumee/prettygreen/node_modules/modernizr/bin/modernizr:121:8)
    at Object.<anonymous> (/home/chf/workspace/mirumee/prettygreen/node_modules/modernizr/bin/modernizr:134:16)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
```

But config file which contains the same configuration works fine (`modernizr-config.json`):

```json
{
  "classPrefix": "",
  "options": [
    "addTest",
    "prefixes",
    "setClasses",
    "testStyles"
  ],
  "feature-detects": [
    "touchevents"
  ]
}
```

and command:

```bash
$ ./node_modules/.bin/modernizr -c modernizr-config.json
Modernizr build saved to modernizr.js
```

**Comment by patrickkettner** [comment]
thanks for the report! total screwup on my part :[ - fixed in master though!

**Comment by odysseuscm** [comment]
It looks like command line also fails for all css tests that have more than one property (where property is an array instead of a string):
e.g.:
$ ./node_modules/.bin/modernizr -f hyphens
$ ./node_modules/.bin/modernizr -f cssgrid

Or maybe I have to write it differently?

**Comment by rejas** [comment]
You are correct @odysseuscm this is indeed broken. I opened a ticket #2479 for this in case you want to work on a fix ;)

**Comment by odysseuscm** [comment]
Thanks. I tried to give it a shot but I understand too little of the inner workings of NodeJS to get to the bottom of it. I did hack my way around it by creating and testing a second object where the property value is an array;
`var obj2 = _.find(source,{property: [prop]});`
But this of course is no fix ...

**Comment by rejas** [comment]
I whipped up a little fix over in #2480 if you want to take a look :-)

**Comment by odysseuscm** [comment]
Thanks, works great. And certainly more elegant than my "solution".
